typescript/infra/scripts/deploy-igp.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  AbacusCore,
+  ChainMap,
+  ChainName,
+  CoreConfig,
+  CoreContracts,
+  MultiProvider,
+  objMap,
+  serializeContracts,
+} from '@abacus-network/sdk';
+
+import { AbacusCoreInfraDeployer } from '../src/core/deploy';
+import { readJSON, writeJSON } from '../src/utils/utils';
+
+import {
+  getCoreContractsSdkFilepath,
+  getCoreEnvironmentConfig,
+  getCoreVerificationDirectory,
+  getEnvironment,
+} from './utils';
+
+class IGPDeployer<
```

Comment:
damn you made a class 😮

Reply:
Well I mostly wanted to inherit from deployer to get the utilities. I would have preferred a functional approach but we are where we are 📦